pkg/diff/util_test.go

```diff
@@ -96,3 +96,32 @@ func (s *testUtilSuite) TestRowToString(c *C) {
 	c.Assert(rowStr, Matches, ".*name: abc.*")
 	c.Assert(rowStr, Matches, ".*info: IsNull.*")
 }
+
+func (s *testUtilSuite) TestMinLenInSlices(c *C) {
+	testCases := []struct {
+		slices [][]string
+		expect int
+	}{
+		{
+			[][]string{
+				{"1", "2"},
+				{"1", "2", "3"},
+			},
+			2,
+		}, {
+			[][]string{
+				{"1", "2"},
+				{},
+			},
```

Comment:
what is the min length of 
```go
[][]string {
	{},
	{"1", "2'},
}
```

Reply:
a mistake, fixed in 5389e91a1bb81777301070c681ff9c1a1d453933